Client/Frontend/Browser/TabManager.swift

```diff
@@ -214,6 +214,10 @@ class TabManager: NSObject {
 
         return nil
     }
+    
+    func getCountForWebsite() -> Int {
+        return allTabs.filter { $0.url?.isWebPage() ?? false && !($0.url?.isAboutHomeURL ?? true) }.count
```

Comment:
`$0.url?.isWebPage() == true && ($0.url?.isAboutHomeURL ?? true)`

Reply:
Changed both expression in amore readable way.

`allTabs.filter { $0.url?.isWebPage() == true && $0.url?.isAboutHomeURL == false }.count`